src/main/haskell/language-kore/test/parser/Data/Kore/Unparser/UnparseTest.hs

```diff
@@ -136,18 +128,15 @@ unparseUnitTests =
                     ]
                 }::KoreDefinition
             )
-            (  "[]\n\n"
-            ++ "    module i\n    endmodule\n    []\n"
-            ++ "    module k\n    endmodule\n    []\n"
-            )
+            "[]\nmodule i\nendmodule\n[]\nmodule k\nendmodule\n[]"
```

Comment:
Could you split this after "\n", as it was before? That way it's much easier to understand what should happen in this test.